specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/textanalytics.json

```diff
@@ -0,0 +1,2903 @@
+{
```

Comment:
Semantic diff to 2022-03-01-preview in main: https://portal.azure-devex-tools.com/tools/diff/eyJzb3VyY2VVcmwiOiJodHRwczovL3Jhdy5naXRodWJ1c2VyY29udGVudC5jb20vQXp1cmUvYXp1cmUtcmVzdC1hcGktc3BlY3MvbWFpbi9zcGVjaWZpY2F0aW9uL2NvZ25pdGl2ZXNlcnZpY2VzL2RhdGEtcGxhbmUvTGFuZ3VhZ2UvcHJldmlldy8yMDIyLTAzLTAxLXByZXZpZXcvdGV4dGFuYWx5dGljcy5qc29uIiwidGFyZ2V0VXJsIjoiaHR0cHM6Ly9yYXcuZ2l0aHVidXNlcmNvbnRlbnQuY29tL0F6dXJlL2F6dXJlLXJlc3QtYXBpLXNwZWNzLzk5MjA5OWRmYmQ1MmZmNWVjNzAwYWNmODlkOWY3OGQ3OWI2MTk4Y2Uvc3BlY2lmaWNhdGlvbi9jb2duaXRpdmVzZXJ2aWNlcy9kYXRhLXBsYW5lL0xhbmd1YWdlL3ByZXZpZXcvMjAyMi0wNS0wMS1wcmV2aWV3L3RleHRhbmFseXRpY3MuanNvbiJ9

Reply:
No changes were detected.